apis/core/lib/domain/queries/table.ts

```diff
@@ -0,0 +1,23 @@
+import { NamedNode } from 'rdf-js'
+import { SELECT } from '@tpluscode/sparql-builder'
+
+import { parsingClient } from '../../query-client'
+import { cc } from '@cube-creator/core/namespace'
+
+export async function * getTablesFromMapping(csvMapping: NamedNode, client = parsingClient) {
```

Comment:
A tiny rename maybe?

```suggestion
export async function * getTablesForMapping(csvMapping: NamedNode, client = parsingClient) {
```